### PR TITLE
Branch reminder date

### PR DESCRIPTION
**Pull request by NeoMinWei**
Fix date output for reminders

### CODE REVIEW

**Review by ryan1604 — APPROVED** [review]
LGTM

File: src/main/java/seedu/financialplanner/commands/DeleteCashflowCommand.java [hunk 1]
```diff
@@ -76,7 +76,8 @@ public DeleteCashflowCommand(RawCommand rawCommand) throws IllegalArgumentExcept
             index = Integer.parseInt(stringIndex);
         } catch (IllegalArgumentException e) {
             logger.log(Level.WARNING, "Invalid argument for index");
-            throw new IllegalArgumentException("Index must be an integer.");
+            throw new IllegalArgumentException("Index must be an integer and be " +
+                    "within the maximum value this program can hold.");
```
Comment (wwweert123) [inline src/main/java/seedu/financialplanner/commands/DeleteCashflowCommand.java:80]: Nice Clarity 👍

File: src/main/java/seedu/financialplanner/storage/LoadData.java [hunk 2]
```diff
@@ -359,8 +363,14 @@ public static HashMap<String, Stock> loadWatchList() {
     }
 
     private static void checkValidInput(double value, int recur) throws FinancialPlannerException {
-        if (value < 0 || recur < 0) {
-            throw new FinancialPlannerException("Amount and number of days cannot be negative");
+        if (value < 0) {
```
Comment (wwweert123) [inline src/main/java/seedu/financialplanner/storage/LoadData.java:366]: 👍

File: src/test/java/seedu/financialplanner/commands/DeleteCashflowCommandTest.java [hunk 3]
```diff
@@ -27,7 +27,8 @@ void testIllegalArgumentException() {
             DeleteCashflowCommand testEntry = new DeleteCashflowCommand(Parser.parseRawCommand("delete a"));
             fail();
         } catch (IllegalArgumentException e) {
-            assertEquals("Index must be an integer.", e.getMessage());
+            assertEquals("Index must be an integer and be within the " +
```
Comment (wwweert123) [inline src/test/java/seedu/financialplanner/commands/DeleteCashflowCommandTest.java:30]: Nice testing

**Review by wwweert123 — APPROVED** [review]
LGTM

**Pull request merged**